### PR TITLE
Collections display fragment

### PR DESCRIPTION
**Pull request by Alec-Shay**
See: https://issues.liferay.com/browse/LPS-111142 and https://issues.liferay.com/browse/LPS-114585

This consolidates the material related to displaying Collections into a new "Displaying Collections" section in Site Building, and completes the user guide-side material related to the Collection Display fragment. More work is still needed for developer-oriented material (including implementing renderers and the information regarding the request attribute).

### CODE REVIEW

File: docs/dxp/7.x/en/site-building/displaying-content/displaying-collections/collection-display-fragments.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+## Displaying a Collection in a Collection Display Fragment
+
+Use the following steps to display a Collection in a Collection Display fragment on any Content Page:
+
+1. Add a collection if you don't already have one. See [Creating Collections](../../../content-authoring-and-management/collections-and-collection-pages/creating-collections.md) for more information.
+
+1. Click the Edit icon (![Edit icon](../../../images/icon-edit.png)) on a Content Page.
+
+1. Click the icon (![Add widget icon](../../../images/icon-add-widget.png)) to open the Fragments and Widgets panel.
```
Comment (sandrodw3) [inline docs/dxp/7.x/en/site-building/displaying-content/displaying-collections/collection-display-fragments.md:13]: I guess this icon is not correct, it's the icon displayed in Widget Pages to add new widgets, but in Content Pages, we have cards2 clay icon for the Fragments and Widgets panel 🙂

File: docs/dxp/7.x/en/site-building/displaying-content/displaying-collections/collection-display-fragments.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Collection Display Fragments
```
Comment (sandrodw3) [inline docs/dxp/7.x/en/site-building/displaying-content/displaying-collections/collection-display-fragments.md:1]: I would put this in singular (Collection Display Fragment)

File: docs/dxp/7.x/en/site-building/displaying-content/displaying-collections/collection-display-fragments.md (hunk 1 above)
Comment (Alec-Shay) [reply docs/dxp/7.x/en/site-building/displaying-content/displaying-collections/collection-display-fragments.md:13]: @sandrodw3  I'm not sure I understand this point of feedback. This looks like the correct icon to me (the icon that appears on the right side on Content Pages). On Widget Pages there isn't an icon there, the panel appears there automatically after clicking to add a widget.

Comment (Alec-Shay) [reply docs/dxp/7.x/en/site-building/displaying-content/displaying-collections/collection-display-fragments.md:13]: Never mind -- the difference was so small it was hard for me to see. I see what you mean now, I'll fix that too. Thanks!

**Comment by Alec-Shay** [comment]
Thanks again! Closing in favor of: https://github.com/sez11a/liferay-learn/pull/274

**Pull request closed without merging**